src/terminal/input/terminalInput.cpp

```diff
@@ -498,16 +498,16 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
         if (ch == UNICODE_NULL)
         {
             // For Alt+Ctrl+Key messages GetCharData() returns 0.
+            // The values of the ASCII characters and virtual key codes
+            // of <Space>, A-Z (as used below) are numerically identical.
             // -> Get the char from the virtual key.
-            ch = LOWORD(MapVirtualKeyW(keyEvent.GetVirtualKeyCode(), MAPVK_VK_TO_CHAR));
+            ch = keyEvent.GetVirtualKeyCode();

```

Comment:
I've taken the liberty to remove this call to `MapVirtualKeyW`.
As far as I now understood virtual key codes in Windows, the numeric values of the vkeys for Space and A-Z are identical with their respective ASCII value. As such a call to `MapVirtualKeyW` is IMO unnecessary.
Am I seing this correctly?

Reply:
I believe you are correct.